### PR TITLE
Update vulnerable deps, cleanup styles and fix precommit hooks

### PR DESCRIPTION
**Pull request by KutsenkoA**

### CODE REVIEW

File: src/api/core/filteringApi.ts [hunk 1]
```diff
@@ -149,6 +149,6 @@ export function toFilteringCriterion<T>(
   filter: IFilteringCriterion<T> | IFilteringCriterionCallback<T>
 ): IFilteringCriterion<T> {
   return typeof filter === "function"
-    ? filter((field) => new FieldFilter(field))
+    ? filter((f) => new FieldFilter(f))
```
Comment (marlosin) [inline src/api/core/filteringApi.ts:152]: why?

File: spec/filtering.ts [hunk 2]
```diff
@@ -4,6 +4,6 @@ import { field } from "../src/api/core/filteringApi";
 export function getRandomFilteringCriteria() {
   return faker.helpers.randomize([
     field("id").isEqualTo("1"),
-    (field: any) => field("someField").isDifferentFrom("someValue"),
+    (col: any) => col("someField").isDifferentFrom("someValue"),
```
Comment (marlosin) [inline spec/filtering.ts:7]: why?

Comment (marlosin) [inline spec/filtering.ts:7]: please enforce typing on `col`

File: src/api/core/filteringApi.ts (hunk 1 above)
Comment (KutsenkoA) [reply src/api/core/filteringApi.ts:152]: shadowed variable 'field', linter is not happy about it :)

File: spec/filtering.ts (hunk 2 above)
Comment (KutsenkoA) [reply spec/filtering.ts]: pointed type a bit better, but I couldn't enforce the whole type. Anyway, it is just a test helper :)

**Pull request merged**